ts/client/src/scripts/mb-admin.ts

```diff
@@ -424,6 +424,8 @@ async function registerPerpMarkets() {
     0.95,
     1.025,
     1.05,
+    1,
```

Comment:
we probably want to edit btc-perp and set some new maintPnlAssetWeight and initPnlAssetWeight

Reply:
also what should be the defaults for this for previously trusted markets?

Reply:
Don't change BTC-PERP yet, we still need to add an instruction for the liquidation of positive pnl that can now take advantage of the difference between the asset weight and 1 to give a liquidation incentive to the liqor.

Defaults: It expresses the risk from unsettled pnl potentially never getting settled. Maybe something like 0.9 / 0.95 for markets we're highly confident in? We need to talk about it.